FluidNC/src/Motors/NullMotor.h

```diff
@@ -2,15 +2,43 @@
 
 #include "MotorDriver.h"
 
+#include <cstdint>
+
 namespace MotorDrivers {
     class Nullmotor : public MotorDriver {
+        int32_t direction = 0;
+        int32_t steps     = 0;
+        int     state     = 0;
+        bool    disabled  = true;
+
     public:
         Nullmotor() = default;
 
         bool set_homing_mode(bool isHoming) { return false; }
 
         bool isReal() override { return false; }
 
+        void step() override {
+            Assert(!disabled, "Cannot step motor while disabled.");
```

Comment:
I think it is unwise to Assert in something that is usually called from an ISR

Reply:
In general, I agree that asserts are "unwise" in an ISR. However, the rationale here was that it should "crash early" if things go wrong. After all, if the ena/dir is incorrect, it will just crash - but then again, if it's wrong and a machine moves there are bigger issues... It was a bit of a trade-off here.